Assignment 1/180501081/occurances.py

```diff
@@ -0,0 +1,6 @@
+s="aabbccddee"
+print("a -> ",s.count('a'))
+print("b -> ",s.count('b'))
+print("c -> ",s.count('c'))
+print("d -> ",s.count('d'))
+print("e -> ",s.count('e'))
```

Comment:
The code should be more dynamic and should work for any character. This will work only for characters a-e.